trestle/common/list_utils.py

```diff
@@ -42,3 +42,15 @@ def is_ordered_sublist(needle: List[str], haystack: List[str]) -> bool:
     needle=['a','b','c'], haystack=['x','y','a','b','z','c'], result = False
     """
     return ' '.join(needle) in ' '.join(haystack)
+
+
+def join_str_to_list_dicts(dict1: Dict, dict2: Dict) -> None:
+    """Join two dicts of str to List into first."""
+    # merge like keys
+    for key in dict1.keys():
+        if key in dict2.keys():
+            dict1[key] = dict1[key] + dict2[key]
```

Comment:
Actually  merge-like can be done in single line with no for:
dict1 = {key: dict1[key] + dict2.get(key, '') for key in dict1}

then merge unlike keys with:
dict1.update({key: dict2[key] for key in dict2 if key not in dict1})

It's somewhat less legible but a lot shorter.

Reply:
Fixed.